frontend/src/components/common/ProfileSidebar.vue

```diff
@@ -62,22 +62,26 @@ const buttonLabel = computed(() => {
             </div>
             <hr class="profile-divider" />
             <p class="options">Options</p>
-            <Button
-                class="sign-out"
-                title="Sign out"
-                aria-expanded="false"
-                aria-label="sign out"
-                :label="buttonLabel"
-                :iconPosition="IconPosition.left"
-                @click="logout"
-                :disabled="loading ? true : false"
-            >
+            <div class="sign-out-wrapper">
                 <Icon
+                    title="Sign out"
                     icon="user--follow"
                     :size="IconSize.small"
                     class="custom-carbon-icon-user--follow"
+                    @click="logout"
+                    :disabled="loading ? true : false"
```

Comment:
just a small question, do we need the icon to click and logout as well?

Reply:
well, might be personal preference but I think it's nice to be able to click the logout or the icon.